server.go

```diff
@@ -3345,9 +3346,14 @@ func (s *Server) DiagnosticsAsRows() []*influxql.Row {
 
 		// Shard may not be local to this node.
 		if sh.store != nil {
-			shardsRow.Columns = append(shardsRow.Columns, "path")
-			shardsRow.Values[0] = append(shardsRow.Values[0], sh.store.Path())
+			if !hasPath {
+				shardsRow.Columns = append(shardsRow.Columns, "path")
+				hasPath = true
+			}
+			index := len(shardsRow.Values) - 1 // Current shardsRow.Values index
+			shardsRow.Values[index] = append(shardsRow.Values[index], sh.store.Path())
```

Comment:
I'm not sure this is right, but I might be missing something. `shardsRow.Values[0]` is a slice -- the code wants to append to this slice, not add another distinct entry to `shardsRow.Values`.

Can you show some raw data to explain what you are trying to fix?